CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
+## 0.4.1
+
+- Require Dart 3.0.
```

Comment:
Might could just bump it to 3.2 – and get new inference magic!

Reply:
sgtm - will do